packages/features/ee/teams/lib/getUserAdminTeams.ts

```diff
@@ -1,96 +1,29 @@
-import type { Prisma } from "@prisma/client";
-
 import { prisma } from "@calcom/prisma";
 import { MembershipRole } from "@calcom/prisma/enums";
 
-export type UserAdminTeams = (Prisma.TeamGetPayload<{
-  select: {
-    id: true;
-    name: true;
-    logoUrl: true;
-    credentials?: true;
-    parent?: {
-      select: {
-        id: true;
-        name: true;
-        logoUrl: true;
-        credentials: true;
-      };
-    };
-  };
-}> & { isUser?: boolean })[];
+export type UserAdminTeams = number[];
 
-/** Get a user's team & orgs they are admins/owners of. Abstracted to a function to call in tRPC endpoint and SSR. */
-const getUserAdminTeams = async ({
-  userId,
-  getUserInfo,
-  getParentInfo,
-  includeCredentials = false,
-}: {
-  userId: number;
-  getUserInfo?: boolean;
-  getParentInfo?: boolean;
-  includeCredentials?: boolean;
-}): Promise<UserAdminTeams> => {
-  const teams = await prisma.team.findMany({
+export const getUserAdminTeams = async (userId: number): Promise<number[]> => {
+  const user = await prisma.user.findFirst({
```

Comment:
queried the user using userId got all his teams and then filter all the teams that have the current user asadmin/owner
(this should be faster becaseu user_id is the primary key for the table so getting all the teams of user should be fast)